bindings/nodejs/CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 1.0.0-rc.2 - 2023-xx-xx
+
+### Changed
+
+- Rename `Account::mintNativeToken` to `createNativeToken`, `Account::increaseNativeTokenSupply` to `mintNativeToken`, `Account::decreaseNativeTokenSupply` to `meltNativeToken`;
+
```

Comment:
Also mention `MintNativeTokenParams` to `CreateNativeTokenParams`

Reply:
Done

Reply:
I don't see it